### ISSUE TITLE
Search inside Talk App doesn't include talk messages

### ISSUE THREAD

**Issue by pschopen** [nextcloud/spreed#11130]
The search inside the talk app (left sidebar) doesn't include the messages inside the chats. The big NC Search (upper right) does. That doesn't make any sense to me.

**Comment by pschopen** [comment]
<< That's okay, it makes a lot of sense for a lot of other people.

When it's on purpose it's fine.

**Comment by pschopen** [comment]
> When it's on purpose it's fine.

Just checked the android app. There the search also searches for messages. Don't you think that it at least should do the same?

**Comment by nickvergessen** [comment]
> Just checked the android app. There the search also searches for messages. Don't you think that it at least should do the same?

No, because the Android app has only one search, while the web interface has a global/unified search in the top right, and the quick conversation filter with unread/mentions options in the top left.

> Also it might make sense to have the chats appearing first and messages appearing underneath? 

That is something we could consider in the future, but not for now.

**Comment by Antreesy** [comment]
> I think we could change the placeholder message to "Filter"

Apart from filtering existing conversations in the list, we're also performing an actual search for open conversations to join and users/groups/circles to create new ones. 
Maybe some of this fit better?
`Filter and search ...`
`Filter and create ...`
`Filter and explore ...`

**Comment by vagner-dias** [comment]
Hello,

It would be beneficial for the NextCloud Talk desktop app to include a search function for messages. Currently, this feature is only available in the web app via "universal search," leaving the desktop app without a similar capability.

**Comment by nickvergessen** [comment]
Yes, we are aware of that and it's our daily pain 😅
